wiki/Help_centre/Upgrading_to_lazer/de.md

```diff
@@ -1,14 +1,12 @@
 ---
 no_native_review: true
-outdated_translation: true
-outdated_since: fd2cdf6390b49d4dfe4648128ba64e996adb985d
 ---
 
 # Umstellung auf lazer
 
-osu!(lazer) ist die nächste große Aktualisierung des Spiels. Es ist der Höhepunkt von mehreren Jahren Arbeit hinter den Kulissen, um das Spiel akribisch neu zu implementieren.
+osu!(lazer) ist das nächste große Update des Spiels. In den letzten Jahren wurde das Spiel von Grund auf neu implementiert.
 
-Das Ziel ist, dass diese Version als Update veröffentlicht wird, das die bestehende stabile Version des Spiels vollständig ersetzt. Es gibt allerdings noch Arbeit zu erledigen, bevor das passieren kann. Am Ende des Tages entscheiden **die Spieler**, wann das geschieht und wir werden die vorherige Version weiter unterstützen, bis die Nutzer migriert sind.
+Diese neue Version soll die bestehende Version des Spiels (Stable) vollständig ersetzen. Es gibt allerdings noch Arbeit zu erledigen, bevor das passieren kann. Am Ende des Tages entscheiden **die Spieler**, wann das geschieht und wir werden die vorherige Version weiter pflegen, bis die Nutzer migriert sind.
```

Comment:
```suggestion
Diese neue Version soll die bestehende Version des Spiels (Stable) vollständig ersetzen. Es gibt allerdings noch Arbeit zu erledigen, bevor das passieren kann. Am Ende des Tages entscheiden **die Spieler**, wann das geschieht und wir werden die vorherige Version weiter unterstützen, bis die Nutzer migriert sind.
```

Maybe it's better to leave it like that :thinking:

`Pflegen` feels more like actively working on it.